crates/pallet-domains/src/staking.rs

```diff
@@ -1872,8 +2121,9 @@ pub(crate) mod tests {
             nominator_id: 0,
             withdraws: vec![(49 * SSC, Ok(()))],
             maybe_deposit: None,
-            expected_withdraw: Some((49 * SSC, false)),
+            expected_withdraw: Some((48999999980000000000, false)),
```

Comment:
Why the change in value here. If there is no reward, then withdrawals should be exact. There should not be any float point loss